src/plugins/sync-plugin.js

```diff
@@ -278,6 +278,23 @@ const restoreRelativeSelection = (tr, relSel, binding) => {
   }
 }
 
+/**
+ * @param {import('prosemirror-state').Transaction} tr
+ * @param {number} pos
```

Comment:
Would it be useful to add a comment explaining what these parameters are and where they come from?

Reply:
I'm sticking to the usual style of Kevin Jahns who doesn't add comments to y-prosemirror in case we need to re-sync with upstream.